Z-Wave controllers: rename to adapter (#39806)
home-assistant/home-assistant.github.io: source/_docs/z-wave/controllers.markdown

---

> title: "Z-Wave adapters"

Reviewer: Should this be in Title Case?

Author: no, we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization), also in titles

description: "Extended instructions how to set up Z-Wave."
related:
  - docs: /integrations/zwave_js/
    title: Z-Wave integration
---

> ## Supported Z-Wave adapters

Reviewer: dito

You need to have a compatible Z-Wave adapter installed. The following devices have been confirmed to work with Z-Wave JS:

{% warning %}

> The firmware of 700 and 800 series Z-Wave adapters have several bugs which impact the stability of the mesh and can cause the adapter to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:

Reviewer: ```suggestion
The firmwares of 700 and 800 series Z-Wave adapters have several bugs which impact the stability of the mesh and can cause the adapter to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
```
I think the plural is correct, since there are several firmwares that are affected. And 700 and 800 series generally have different firmwares.

{% warning %}

The firmwares of 700- and 800-series Z-Wave adapters have several bugs which impact the stability of the mesh and can cause the adapter to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:

> - 700-series:

Reviewer: Silicon Labs actually call it "700 series" and "800 series": https://www.silabs.com/wireless/z-wave/introduction-to-z-wave-800-series

  - Z-Wave.Me Razberry 2 (500 series)
  - Z-Wave.Me Razberry 1 (300 series)

If you are just starting out, we recommend that you purchase a 800-series controller (with firmware updated to >=7.23.2). The 800-series adapters are the most future proof and have the best RF performance.

{% tip %}
It's recommended to use a USB stick, not a module. Passing a module through Docker is more complicated than passing a USB stick through.
{% endtip %}

> ## Third-party hubs

Reviewer: Also, Title Case?

For best experience, it is recommended to use an adapter directly with Home Assistant. If this doesn't work for you, you can use a hub that supports Z-Wave. Home Assistant supports the following third-party hubs with Z-Wave support:

- [Vera](/integrations/vera/)
- [Fibaro](/integrations/fibaro/)
- [SmartThings](/integrations/smartthings/)
- [Z-Wave.Me Z-Way](/integrations/zwave_me)

> ## Adapter notes

Reviewer: Title Case

  - Z-Wave.Me Razberry 2 (500 series)

>   - Z-Wave.Me Razberry 1 (300 series)

Reviewer: I cannot select the headline, but if we're throwing USB sticks and GPIO modules under the generic term "adapter" now, should we then merge the Raspberry pi modules with the corresponding sections, since they are technically also 500/700/800 series?

Author: hm, but its still nice to have them separated by form factor? so if I don't care about the modules, they don't clutter up the list?
Or is it unlikely that people choose by form factor?

Reviewer: In that case, should the other 3 headlines instead be "XXX series **USB** adapters"?

> If you are just starting out, we recommend that you purchase a 700 series adapter or a Raspberry Pi module. The 700 series adapters are the more recent version (when compared to the 500 series). The 700 series adapters support SmartStart, which allows you to add a device by scanning a QR code.

Reviewer: Dito, do we need the distinction here?

> If you are just starting out, we recommend that you purchase a 700 series adapter or a Raspberry Pi module. The 700 series adapters are the more recent version (when compared to the 500 series). The 700 series adapters support SmartStart, which allows you to add a device by scanning a QR code.

Reviewer: Actually, I think this entire sentence needs to be reworked:

- We recommend starting with 800 series now, especially in EU because of the upcoming Long Range support
- 700 series only supports Long Range in US (with one exception)
- 700 and 800 series (and some recent 500 series firmwares) support SmartStart

But this is too complex to explain here, so we should just say that 800 series is the most future proof and has the best RF performance.

> If you are just starting out, we recommend that you purchase a 700 series adapter or a Raspberry Pi module. The 700 series adapters are the more recent version (when compared to the 500 series). The 700 series adapters support SmartStart, which allows you to add a device by scanning a QR code.

Author: ```suggestion
If you are just starting out, we recommend that you purchase a 700 series adapter. The 700 series adapters are the more recent version (when compared to the 500 series). The 700 series adapters support SmartStart, which allows you to add a device by scanning a QR code.
```

{% tip %}
It's recommended to use a USB stick, not a module. Passing a module through Docker is more complicated than passing a USB stick through.
{% endtip %}

> ## Stick or module alternatives

Reviewer: Should we just say "3rd party hubs"?

> ## Stick or module alternatives

Author: ```suggestion
## Third pary hubs
```

> The alternative to a stick or module is a hub that supports Z-Wave. Home Assistant supports the following hubs with Z-Wave support:

Reviewer: Should we hint that using an adapter with HA directly (and not a 3rd party hub) is recommended for the best experience?

> The alternative to a stick or module is a hub that supports Z-Wave. Home Assistant supports the following hubs with Z-Wave support:

Author: ```suggestion
For best experience, it is recommended to use an adapter directly with Home Assistant. If this doesn't work for you, you can use a hub that supports Z-Wave. Home Assistant supports the following hubs with Z-Wave support:
```

- [Vera](/integrations/vera/)
- [Fibaro](/integrations/fibaro/)
- [SmartThings](/integrations/smartthings/)
- [Z-Wave.Me Z-Way](/integrations/zwave_me)

## Adapter notes

### 800 Series Adapters

> Z-Wave JS does not support Z-Wave Long Range yet.

Reviewer: This is no longer true.

Author: Thank you for the catch. Removed!
